indico/htdocs/js/indico/widgets/protection_widget.js

```diff
@@ -49,9 +50,16 @@
                         }
                     });
                 }
+                $aclListField.toggleClass('hidden', !isProtected);
+                $enableAclLink.toggleClass('hidden', isProtected);
             }
         });
 
+        $enableAclLink.on('click', function(evt) {
+            evt.preventDefault();
+            $('input[value="protected"]').trigger('click');
```

Comment:
Quite generic. As there's no decent ID selector I'd go for `$('input[name=protection_mode][value=protected]')`